src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/watasks/controllers/PostTaskInitiateByIdControllerTest.java

```diff
@@ -274,6 +279,82 @@ public void should_return_a_201_when_initiating_a_specific_access_task_by_id() {
+            new TaskAttribute(TASK_TYPE, "processApplication"),
+            new TaskAttribute(TASK_NAME, "process Application"),
+            new TaskAttribute(TASK_CASE_ID, taskVariables.getCaseId()),
+            new TaskAttribute(TASK_TITLE, "process Application"),
+            new TaskAttribute(TASK_CREATED, formattedCreatedDate),
+            new TaskAttribute(TASK_DUE_DATE, formattedDueDate)
+        ));
+
+        Response result = restApiActions.post(
+            ENDPOINT_BEING_TESTED,
+            taskId,
+            req,
+            caseworkerCredentials.getHeaders()
+        );
+
+        //Note: this is the TaskResource.class
+        result.prettyPrint();
+
+        result.then().assertThat()
+            .statusCode(HttpStatus.CREATED.value())
+            .and()
+            .body("task_id", equalTo(taskId))
+            .body("task_name", equalTo("process Application"))
+            .body("task_type", equalTo("processApplication"))
+            .body("state", equalTo("UNASSIGNED"))
+            .body("task_system", equalTo("SELF"))
+            .body("security_classification", equalTo("PUBLIC"))
+            .body("title", equalTo("process Application"))
+            .body("created", notNullValue())
+            .body("due_date_time", notNullValue())
+            .body("auto_assigned", equalTo(false))
+            .body("has_warnings", equalTo(false))
+            .body("case_id", equalTo(taskVariables.getCaseId()))
+            .body("case_type_id", equalTo("WaCaseType"))
+            .body("jurisdiction", equalTo("WA"))
+            .body("region", equalTo("1"))
+            .body("location", equalTo("765324"))
+            .body("location_name", equalTo("Taylor House"))
+            .body("execution_type_code.execution_code", equalTo("CASE_EVENT"))
+            .body("execution_type_code.execution_name", equalTo("Case Management Task"))
+            .body(
+                "execution_type_code.description",
+                equalTo("The task requires a case management event to be executed by the user. "
+                            + "(Typically this will be in CCD.)")
+            )
+            .body("work_type_resource.id", equalTo("hearing_work"))
+            .body("work_type_resource.label", equalTo("Hearing work"))
+            .body("role_category", equalTo("LEGAL_OPERATIONS"))
+            .body("description", equalTo("[Decide an application](/case/WA/WaCaseType/${[CASE_REFERENCE]}/"
+                                             + "trigger/decideAnApplication)"))
+            .body("task_role_resources.size()", equalTo(10))
+            .body("additional_properties", equalToObject(Map.of(
+                "key1", "value1",
+                "key2", "value2",
+                "key3", "value3",
+                "key4", "value4"
+            ))).body("minor_priority", equalTo(500))
+            .body("major_priority", equalTo(1000))
+            .body("priority_date", notNullValue());
```

Comment:
as well as notNullValue, should we also assert that it is set to the taskAttributes.dueDate?